snippets/c.snippets

```diff
@@ -125,11 +125,68 @@ snippet do
 ##
 ## Functions
 # function definition
-snippet fun
-	${1:void} ${2:function_name}(${3})
+#snippet fun
+#	${1:void} ${2:function_name}(${3})
+#	{
+#		${4}
+#	}
```

Comment:
Yes, think it make sense to leave it for backward-compatibility. Do you prefer name fun0, fun2 etc or we can add dfun0, dfun1 version for documentation purpose. Thus, we can leave funN versions without documentation and add dfunN version to include documentation. I personally, think that the present documentation block should slightly force people provide some explanation to the instance.